RFCs/FS-1128-better-interpolated-triple-quoted-strings.md

```diff
@@ -21,99 +22,117 @@ let json = $$"""
 };
 """
 ```
+
 In the above example, `$$` starting the string literal indicates that interpolated expression will be enclosed in `{{...}}` and single `{` and `}` will be treated as content.
 
 # Motivation
 
-Triple quoted strings are already quite handy for embedding e.g. XML in string literals, but not as convenient for embedding some other languages that use `{` and `}` in their syntax, e.g. JSON or CSS. Currently every occurrence of a curly brace in an interpolated string needs to be escaped, which can quickly become tedious. With this extended syntax, only the braces around interpolated expression need special attention, and that is where programmer's focus already is.
+Triple quoted strings are already quite handy for embedding e.g. XML in string literals, but not as convenient for embedding some other languages that use `{` and `}` in their syntax, e.g. JSON or CSS.
+Currently every occurrence of a curly brace in an interpolated string needs to be escaped, which can quickly become tedious.
+With this extended syntax, only the braces around interpolated expression need special attention, and that is where programmer's focus already is.
 
 The motivation is not to enable any novel use cases, but to make working with interpolated string literals a much more seamless experience in the use cases mentioned above.
 
 This change should:
+
 1. be backwards compatible with existing support for triple quoted strings
-2. align with C# raw string literals reasonably well
+2. align with C# [raw string literals](https://github.com/dotnet/csharplang/blob/main/proposals/csharp-11.0/raw-string-literal.md) reasonably well
 
 # Detailed design
 
-The existing form of interpolated triple quoted strings: `$"""...{}..."""` is extended to allow multiple dollar signs at the begining of the string literal. The count of these initiatory `$` characters indicates how many `{` and `}` characters are used to delimit interpolation expression within the content of the literal.
+The existing form of interpolated triple quoted strings: `$"""...{}..."""` is extended to allow multiple dollar signs at the begining of the string literal.
+The count of these initiatory `$` characters indicates how many `{` and `}` characters are used to delimit interpolation expression within the content of the literal.
```

Comment:
"initiatory"? Looks like it's a word. I didn't know it :).

Reply:
Tbh now I see that this sentence would preserve its meaning and be just as clear without it, so I can just remove the suspicious word